### PR TITLE
(C) Mg/feature branch example

### PR DESCRIPTION
**Pull request by calopter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Person B should modify content here.
 
 ## Person C:
 
-Person C should modify content here.
+Person C modified content here.
```
Comment (stupendousC) [inline README.md:15]: hey you're supposed to write something here lol

File: uh.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+favorite-color
```
Comment (stupendousC) [inline uh.txt:1]: what is it though? lol

**Review by stupendousC — CHANGES_REQUESTED** [review]
so many questions...

**Comment by calopter** [comment]
shipit

File: uh.txt (hunk 2 above)
Comment (calopter) [reply uh.txt:1]: green

**Comment by stupendousC** [comment]
i guess i'll approve it, sigh...

**Pull request merged**